src/azure-cli/azure/cli/command_modules/monitor/validators.py

```diff
@@ -298,6 +298,8 @@ def process_action_group_detail_for_creation(namespace):
         'azure_function_receivers': [r for r in receivers if isinstance(r, AzureFunctionReceiver)],
         'tags': ns.get('tags') or None
     }
+    if hasattr(namespace, 'tags'):
+        del namespace.tags
```

Comment:
Why should the tags attribute of namespace be deleted?

Reply:
In track2, only necessary parameters can be passed to `kwargs`; otherwise track2 SDK will throw error.
In this case `tag` should be set in payload instead of in kwargs`, you can see it is already set in `action_group_resource_properties` (payload).

---
In reply to: [559293145](https://github.com/Azure/azure-cli/pull/16551#discussion_r559293145) [](ancestors = 559293145)